agent/internal/agent.go

```diff
@@ -36,6 +38,33 @@ type agent struct {
 	cm     *actor.Ref
 }
 
+func (a *agent) addProxy(config *container.Config) {
+	addVars := map[string]string{
+		"HTTP_PROXY": a.Options.HTTPProxy, "HTTPS_PROXY": a.Options.HTTPSProxy,
```

Comment:
This is pretty nitpicky, but if `make fmt` allows you to set these each on their own line, I find that more consistent with most of our code.  If not that's fine.